Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
```

Comment:
Add a description for each task to improve clarity, especially for developers who are not familiar with the project.